src/components/activity/ActivityController.cpp

```diff
@@ -0,0 +1,34 @@
+#include "components/activity/ActivityController.h"
+
+using namespace Pinetime::Controllers;
+
+ActivityController::ActivityController(Controllers::DateTime& dateTimeController, Controllers::Settings& settingsController)
+  : dateTimeController {dateTimeController}, settingsController {settingsController} {
+}
+
+bool ActivityController::UpdateSteps(uint32_t nbSteps) {
```

Comment:
This function is very obscure. It's called UpdateSteps, yet it returns a value based on whether there's been enough activity recently. You'd have to examine the entire function thoroughly to figure that out.

Reply:
Fixed. Sorry about that, I just started writing without thinking about the structure first.